client/reader/stream/featured-asset.jsx

```diff
@@ -1,152 +0,0 @@
-/**
-* External dependencies
-*/
-import React from 'react';
-import ReactDom from 'react-dom';
-import { assign, constant, throttle } from 'lodash';
-
-/**
-* Internal dependencies
-**/
-import EmbedHelper from 'reader/embed-helper';
-
-const maxWidth = () => Math.min( 720, window.innerWidth );
-
-export default class FeaturedAsset extends React.Component {
-	constructor( props ) {
-		super( props );
-		this.state = {
-			suppressFeaturedImage: false,
-			useFeaturedEmbed: props.useFeaturedEmbed
-		};
-
-		[	'handleImageError',
-			'updateFeatureSize',
-			'getMaxFeaturedWidthSize',
-			'setEmbedSizingStrategy',
-			'handleResize'
-		].forEach( method => {
-			this[ method ] = this[ method ].bind( this );
-		} );
-
-		this.setEmbedSizingStrategy( props.featuredEmbed );
-		this.setImageSizingStrategy( props.featuredImage );
-	}
-
-	componentWillReceiveProps( nextProps ) {
-		// if the next set of props
-		if ( nextProps.featuredImage !== this.props.featuredImage ) {
-			this.setState( {
-				suppressFeaturedImage: false,
-				useFeaturedEmbed: nextProps.useFeaturedEmbed
-			} );
-			this.setImageSizingStrategy( nextProps.featuredImage );
-		}
-
-		if ( nextProps.featuredEmbed !== this.props.featuredEmbed ) {
-			this.setEmbedSizingStrategy( nextProps.featuredEmbed );
-		}
-	}
-
-	componentWillMount() {
-		this._unmounted = false;
-	}
-
-	componentWillUnmount() {
-		this._unmounted = true;
-		window.removeEventListener( 'resize', this._handleResize );
-	}
-
-	componentDidMount() {
-		this.updateFeatureSize();
-		this._handleResize = throttle( this.handleResize, 100 );
-		window.addEventListener( 'resize', this._handleResize );
-	}
-
-	handleResize() {
-		this.updateFeatureSize();
-	}
-
-	getMaxFeaturedWidthSize() {
-		return ReactDom.findDOMNode( this ).parentNode.offsetWidth;
-	}
-
-	setEmbedSizingStrategy( featuredEmbed ) {
-		let sizingFunction = constant( {} );
-		if ( featuredEmbed ) {
-			const embedSize = EmbedHelper.getEmbedSizingFunction( featuredEmbed );
-			sizingFunction = ( available = this.getMaxFeaturedWidthSize() ) => embedSize( available );
-		}
-		this.getEmbedSize = sizingFunction;
-	}
-
-	setImageSizingStrategy( featuredImage ) {
```

Comment:
So I'm pretty sure this is referencing an old unused file from the prefresh.  We still probably have some duplicated deleteable files located within the `/reader` directory that we have since migrated to `/blocks`.  

If you look at the `FeaturedVideo` file you are importing, it has all of the sizing logic within!

So this is all 👍